### DISCUSSION TITLE
Mixed Size Option in One Button Run and Upscale

### DISCUSSION THREAD

**Discussion by pto2k** [AIrjen/OneButtonPrompt#96] [Ideas]
Hi,

With Quality Gate in OBRaU, I can try to get the best of one generated prompt. 
However, one prompt is fixed with one size. 

I think it would be great to have a new 'Mixed in QG' size option.
I imagine that is just like the 'All' option except it will also randomly select different size options during Quality Gate generations of same prompt.

I think this could help to find more variations with same prompt because aspect ratio has such a big impact on the layout. Sometime I find it gives very different output on same seed.

PS: another 'ultra tall' option would also be fun to have

**Comment by AIrjen** [comment]
Yup yup yup! Having some ideas about this. I just needed to update the sizings as well, since we have SD XL coming out, running on 1024 base size ratio.

I am also adding a "wild" mode ;)

**Reply by AIrjen** [reply]
Done! You can get it in the latest version.

Now, the sizes "all" and "wild" will both change sizings in the Quality Gate.

Have fun!

**Reply by pto2k** [reply]
Wonderful!
